lib/id3c/cli/command/manifest.py

```diff
@@ -483,3 +542,12 @@ def load_ndjson(file: Iterable[str]) -> Iterable:
     Load newline-delimited JSON records from *file*.
     """
     yield from (json.loads(line) for line in file)
+
+
+def find_violating_column_names(extra_columns: List[Tuple[str, Union[str, dict]]]):
+    """
+    Find any reserved column names that also appeaer in extra_columns.
+    """
+    reserved_columns_list = []
```

Comment:
the `reserved_columns_list` variable is immediately overwritten in the following line, making this line unnecessary.

Reply:
Removed!